AdminServer/appscale/admin/__init__.py

```diff
@@ -266,6 +241,25 @@ def version_from_payload(self):
       raise CustomHTTPError(HTTPCodes.BAD_REQUEST,
                             message='Invalid version ID')
 
+    if 'basicScaling' in version or 'manualScaling' in version:
+      raise CustomHTTPError(HTTPCodes.BAD_REQUEST,
+                            message='Only automaticScaling is supported')
+
+    # Create a revision ID to differentiate between deployments of the same
+    # version.
+    version['revision'] = int(time.time() * 1000)
```

Comment:
I figured milliseconds would be unique enough to prevent collisions between versions, but I don't have anything against multiplying it by a million.